server/app/views/questiontypes/MapQuestionFragment.html

```diff
@@ -24,51 +24,53 @@
   ></div>
 
   <!-- Filters -->
-  <fieldset class="grid-row grid-gap">
-    <legend
-      class="usa-legend text-bold"
-      th:text="#{map.filterLegendText}"
-    ></legend>
-    <div
-      th:each="filter, i : ${mapQuestion.getFilters()}"
-      class="grid-col flex-align-self-end"
-    >
-      <label
-        class="usa-label"
-        th:for="'filter' + ${i.index + 1} + '-' + ${mapId}"
-        th:utext="${filter.formattedSettingDisplayName(ariaLabelForNewTabs)}"
-      ></label>
-      <select
-        class="usa-select"
-        th:data-filter-key="${filter.settingValue()}"
-        th:id="'filter' + ${i.index + 1} + '-' + ${mapId}"
-        th:data-map-id="${mapId}"
+  <div th:if="${mapQuestion.getFilters().size() > 0}">
```

Comment:
```suggestion
  <th:block th:if="${mapQuestion.getFilters().size() > 0}">
```

(non-blocking, fyi) If you need to apply some Thymeleaf construct, but don't want to add markup use `th:block` which creates what Thymeleaf calls a synthetic element. It doesn't get rendered and is only there for Thymeleaf processing.

Reply:
I like it! Always forget about this